microcosm_elasticsearch/tests/fixtures.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test fixtures.
+
+"""
+from elasticsearch_dsl import Q, Text
+from microcosm.api import binding
+
+from microcosm_elasticsearch.indexing import make_index
+from microcosm_elasticsearch.models import Model
+from microcosm_elasticsearch.store import Store
+
+
+@binding("example_index")
+def create_example_index(graph):
+    return make_index(graph)
+
+
+class Person(Model):
```

Comment:
FYI: `String` is deprecated; use `Text` or `Keyword`. Also `UUID` isn't supported really.